jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -189,15 +188,15 @@ private Configuration(
 
     statsFactory = new StatsFactoryImpl(new NullStatsReporter());
 
-    this.disableGlobalTracer = disableGlobalTracer;
+    this.enableGlobalTracer = enableGlobalTracer;
   }
 
   public static Configuration fromEnv() {
     return new Configuration(
         getProperty(JAEGER_SERVICE_NAME),
         SamplerConfiguration.fromEnv(),
         ReporterConfiguration.fromEnv(),
-        getPropertyAsBool(JAEGER_DISABLE_GLOBAL_TRACER));
+        getPropertyAsBool(JAEGER_ENABLE_GLOBAL_TRACER));
```

Comment:
Why are we reversing the defaults?

Reply:
We'd rather have this not register the local tracer as the global tracer, so eagerly. If someone wishes to have this feature, they should be made aware of it, and they can set the config appropriately.